content/docs/learning-guide/tutorials/environments/create-terrain-from-images/create-terrain-shape.md

```diff
@@ -0,0 +1,159 @@
+
+5. To generate terrain that is the same resolution as the heightmap, in the **Axis Aligned Box Shape** component, set the value of the **Dimensions:** **X** and **Y** properties to the resolution (in pixels) of the heightmap. The **Dimensions: Z** property can be used to scale the elevation. Enter a value for the **Dimensions: Z** property that roughly represents the maximum desired elevation in meters. Our tutorial heightmap image is 256 x 256 pixels and represents a 50 meter change in elevation, so set the dimensions to `(256.0, 256.0, 50.0)`. If you wish to use a different heightmap, set the dimensions as appropriate for that heightmap's resolution.
+
+    {{< tip >}}
+For the best results, the **X** and **Y** dimensions of the **Axis Aligned Box Shape** should be set to the heightmap pixel size multiplied by the **Height Query Resolution**. This ensures that there is exactly one heightmap pixel for each world unit used by the terrain system. When the **Height Query Resolution** is set to 1 meter (the default), the box size in meters should be exactly the same as the heightmap pixel resolution. If the box dimensions are too small, some heightmap pixels won't get used, which can cause unexpected smooth areas or areas of excessively sharp elevation changes.
+    {{< /tip >}}
+
+6. The **Axis Aligned Box Shape** is centered around the entity's position, so with the entity at `(0.0, 0.0, 0.0)` the box extends from `-25.0` to `25.0` on the Z axis. However, by default the minimum terrain elevation is set to `0.0`. The bottom of the box shape needs to be at least at the minimum terrain elevation value on the world Z axis because any terrain below the minimum elevation will be clipped. In the Terrain Spawner entity, in the **Transform** component, set the **Translate: Z** property to a value that is half of the value you used for the **Dimensions: Z** property in the previous step. For example, if you entered `50.0` in the previous step, use `25.0` for the **Translate: Z** property to move the entity up so that the bottom of the box is at `0.0` on the world Z axis and the top of the box extends to `50.0`.
+
+    {{< tip >}}
+Once you have set the dimensions of the box shape, you can hide it by disabling the **Filled** property in the **Axis Aligned Box Shape** component.
+    {{< /tip >}}
+
+7. In Entity Inspector, add a [**Terrain Height Gradient List**](/docs/user-guide/components/reference/terrain/height_gradient_list) component to the **Terrain Spawner** entity. This component will reference the gradient entity in a later step.
+
+The following image is the complete **Terrain Spawner** entity:
+
+    {{< image-width src="/images/learning-guide/tutorials/environments/terrain-from-images/terrain-spawner-entity.png" width="450" alt="The completed Terrain Spawner entity." >}}
```

Comment:
This image didn't show up for me on the deploy preview for some reason

Reply:
I'm not sure what's up with this? It renders correctly for me locally, and the image itself is in the deployment: https://deploy-preview-2146--o3deorg.netlify.app/images/learning-guide/tutorials/environments/terrain-from-images/terrain-spawner-entity.png

If it breaks in the actual deployment, I can submit a fix that foregoes the use of the image-width shortlink. I've seen the shortlink have other problems when used inside of a tab shortlink, so it might be another compatibility problem between the two that just isn't reproducing locally for some reason.

I'd _prefer_ to keep the image-width link if possible though, because it has the added benefit of making the image clickable to display it larger in its own browser tab.

Reply:
Sounds like a plan, yeah the markup for the image looks fine so I'm not sure why it didn't work on the deploy, hopefully it should work fine once it's actually merged.